src/main/java/org/broadinstitute/hellbender/utils/test/SamAssertionUtils.java

```diff
@@ -377,15 +445,20 @@ public static void assertEqualBamFiles(
      */
     public static void assertCRAMContentsIfCRAM(final File putativeCRAMFile) {
         if (IOUtils.isCramFile(putativeCRAMFile)) {
-            assertCRAMContents(putativeCRAMFile);
+            assertCRAMContents(putativeCRAMFile.toPath());
+        }
+    }
+    public static void assertCRAMContentsIfCRAM(final Path putativeCRAMPath) {
```

Comment:
javadoc for this method

Reply:
Done; I also changed the File version to more cleanly call the Path version.